src/Commands/UserCommands/HelpCommand.php

```diff
@@ -69,11 +69,16 @@ public function execute()
             );
 
             foreach ($command_objs as $command) {
+                if($command->showInHelp()) {
```

Comment:
Not sure you tested this one, give it a go and see if this does what you want... 😉 
Also, there should be a space after the `if` like so: `if (...)`